### DISCUSSION TITLE
Stuck in boot menu when booting in KVM

### DISCUSSION THREAD

**Discussion by Disil** [SpiralLinux/SpiralLinux-project#332] [Q&A] [answered]
I downloaded the latest KDE iso (12.231008) and it couldnt boot using virt-manager. It does boot to the boot menu, but when I click the boot menu, neither of them worked (the normal and failsafe). It just blank the screen for a second, and back to the boot menu. 

I'll be happy to provide any info from the log.

**Comment by geckolinux** [comment]
Hi there, did you verify the checksum of the ISO?

**Comment by Disil** [comment, marked as answer]
Bruh, my bad. I verified them and the checksum does not match. Will re-download later, and I'll mark this issue completed after that.

**Reply by geckolinux** [reply]
Great, thanks for confirming.
